game.html

```diff
@@ -26,59 +26,36 @@
 	</section> 
```

Comment:
The very first thing I notice here is the lack of proper spacing and use of line breaks. Double check that your Sublime settings are using two spaced tabs for your HTML, CSS and JS. 

I'm going to make some quick comments below in a few areas where you should also remove white space.